2020-10-29 13:02:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "
    DROP TABLE IF EXISTS ORDER[*] CASCADE "; se esperaba "identifier"
Syntax error in SQL statement "
    DROP TABLE IF EXISTS ORDER[*] CASCADE "; expected "identifier"; SQL statement:

    drop table if exists order CASCADE  [42001-200]
2020-10-29 13:02:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "
    CREATE TABLE ORDER[*] (
       ID INTEGER NOT NULL,
        NAME VARCHAR(255),
        PRIMARY KEY (ID)
    )"; se esperaba "identifier"
Syntax error in SQL statement "
    CREATE TABLE ORDER[*] (
       ID INTEGER NOT NULL,
        NAME VARCHAR(255),
        PRIMARY KEY (ID)
    )"; expected "identifier"; SQL statement:

    create table order (
       id integer not null,
        name varchar(255),
        primary key (id)
    ) [42001-200]
2020-11-06 18:31:08 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'' (PET: ""ID"" INTEGER NOT NULL)"
Data conversion error converting "'' (PET: ""ID"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."PET"("ID","NAME","PHOTOURL","PRICE","STATUS","CATEGORY_ID","TAG_ID")VALUES(?,?,?,?,?,?,?) [22018-200]
2020-11-06 18:31:08 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'' (PET: ""ID"" INTEGER NOT NULL)"
Data conversion error converting "'' (PET: ""ID"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."PET"("ID","NAME","PHOTOURL","PRICE","STATUS","CATEGORY_ID","TAG_ID")VALUES(?,?,?,?,?,?,?) [22018-200]
2020-11-06 18:32:06 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'' (PET: ""CATEGORY_ID"" INTEGER)"
Data conversion error converting "'' (PET: ""CATEGORY_ID"" INTEGER)"; SQL statement:
INSERT INTO "PUBLIC"."PET"("ID","NAME","PHOTOURL","PRICE","STATUS","CATEGORY_ID","TAG_ID")VALUES(?,?,?,?,?,?,?) [22018-200]
2020-11-06 18:32:06 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'' (PET: ""CATEGORY_ID"" INTEGER)"
Data conversion error converting "'' (PET: ""CATEGORY_ID"" INTEGER)"; SQL statement:
INSERT INTO "PUBLIC"."PET"("ID","NAME","PHOTOURL","PRICE","STATUS","CATEGORY_ID","TAG_ID")VALUES(?,?,?,?,?,?,?) [22018-200]
2020-11-06 18:32:18 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'' (PET: ""PRICE"" FLOAT NOT NULL)"
Data conversion error converting "'' (PET: ""PRICE"" FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."PET"("ID","NAME","PHOTOURL","PRICE","STATUS","CATEGORY_ID","TAG_ID")VALUES(?,?,?,?,?,?,?) [22018-200]
2020-11-06 18:32:18 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'' (PET: ""PRICE"" FLOAT NOT NULL)"
Data conversion error converting "'' (PET: ""PRICE"" FLOAT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."PET"("ID","NAME","PHOTOURL","PRICE","STATUS","CATEGORY_ID","TAG_ID")VALUES(?,?,?,?,?,?,?) [22018-200]
2020-11-06 18:32:43 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'' (PET: ""STATUS"" BOOLEAN NOT NULL)"
Data conversion error converting "'' (PET: ""STATUS"" BOOLEAN NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."PET"("ID","NAME","PHOTOURL","PRICE","STATUS","CATEGORY_ID","TAG_ID")VALUES(?,?,?,?,?,?,?) [22018-200]
2020-11-06 18:32:43 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'' (PET: ""STATUS"" BOOLEAN NOT NULL)"
Data conversion error converting "'' (PET: ""STATUS"" BOOLEAN NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."PET"("ID","NAME","PHOTOURL","PRICE","STATUS","CATEGORY_ID","TAG_ID")VALUES(?,?,?,?,?,?,?) [22018-200]
2020-11-06 18:33:03 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'' (PET: ""CATEGORY_ID"" INTEGER)"
Data conversion error converting "'' (PET: ""CATEGORY_ID"" INTEGER)"; SQL statement:
INSERT INTO "PUBLIC"."PET"("ID","NAME","PHOTOURL","PRICE","STATUS","CATEGORY_ID","TAG_ID")VALUES(?,?,?,?,?,?,?) [22018-200]
2020-11-06 18:33:03 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'' (PET: ""CATEGORY_ID"" INTEGER)"
Data conversion error converting "'' (PET: ""CATEGORY_ID"" INTEGER)"; SQL statement:
INSERT INTO "PUBLIC"."PET"("ID","NAME","PHOTOURL","PRICE","STATUS","CATEGORY_ID","TAG_ID")VALUES(?,?,?,?,?,?,?) [22018-200]
2020-11-13 14:08:36 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "HIBERNET_SEQUENCE" no encontrada
Table "HIBERNET_SEQUENCE" not found; SQL statement:
SELECT * FROM HIBERNET_SEQUENCE [42102-200]
2020-11-13 17:22:48 jdbc[25]: null
org.h2.message.DbException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:245)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:186)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:358)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:477)
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:725)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:711)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:622)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 18 more
2020-11-13 20:54:55 jdbc[45]: null
org.h2.message.DbException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:245)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:186)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:358)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:477)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:560)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:112)
	at org.springframework.boot.autoconfigure.h2.H2ConsoleAutoConfiguration.lambda$h2Console$0(H2ConsoleAutoConfiguration.java:73)
	at org.springframework.beans.factory.ObjectProvider.ifAvailable(ObjectProvider.java:93)
	at org.springframework.boot.autoconfigure.h2.H2ConsoleAutoConfiguration.h2Console(H2ConsoleAutoConfiguration.java:72)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:650)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:635)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1336)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1176)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:556)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:516)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:324)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:322)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:207)
	at org.springframework.boot.web.servlet.ServletContextInitializerBeans.getOrderedBeansOfType(ServletContextInitializerBeans.java:211)
	at org.springframework.boot.web.servlet.ServletContextInitializerBeans.getOrderedBeansOfType(ServletContextInitializerBeans.java:202)
	at org.springframework.boot.web.servlet.ServletContextInitializerBeans.addServletContextInitializerBeans(ServletContextInitializerBeans.java:96)
	at org.springframework.boot.web.servlet.ServletContextInitializerBeans.<init>(ServletContextInitializerBeans.java:85)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.getServletContextInitializerBeans(ServletWebServerApplicationContext.java:255)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.selfInitialize(ServletWebServerApplicationContext.java:229)
	at org.springframework.boot.web.embedded.tomcat.TomcatStarter.onStartup(TomcatStarter.java:53)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5128)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1384)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1374)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
	at java.util.concurrent.AbstractExecutorService.submit(Unknown Source)
	at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:909)
	at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:843)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1384)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1374)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
	at java.util.concurrent.AbstractExecutorService.submit(Unknown Source)
	at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:909)
	at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:262)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
	at org.apache.catalina.core.StandardService.startInternal(StandardService.java:421)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
	at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:930)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
	at org.apache.catalina.startup.Tomcat.start(Tomcat.java:486)
	at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.initialize(TomcatWebServer.java:123)
	at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.<init>(TomcatWebServer.java:104)
	at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getTomcatWebServer(TomcatServletWebServerFactory.java:437)
	at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getWebServer(TomcatServletWebServerFactory.java:191)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.createWebServer(ServletWebServerApplicationContext.java:178)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.onRefresh(ServletWebServerApplicationContext.java:158)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:545)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:143)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:758)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:750)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1237)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at com.upemor.petstorerest.PetstorerestApplication.main(PetstorerestApplication.java:10)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.boot.devtools.restart.RestartLauncher.run(RestartLauncher.java:49)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:622)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 82 more
